packages/docs/docs/guide/markdown.md

```diff
@@ -275,6 +275,46 @@ export default {
 }
 ```
 
+In addition to a single line, you can also specify multiple lines, line ranges, or both.
+
+- Line ranges: e.g. `{5-8}`, `{3-10}`, `{10-17}`
+- Multiple single lines: e.g. `{4,7,9}`
+- Line ranges and single lines: e.g. `{4,7-13,16,23-27,40}`
+
+**Input**
+
+````
+``` js{1,4,6-7}
+export default { // Highlighted
```

Comment:
When I was doing the translation, I was confused by `// Highlighted`. What does this stand for? Not only this line is highlighted.

Reply:
I added that just as an indication that line would also be highlighted.

The other lines all have something saying they're highlighted (line 4 literally says that and lines 6 says lines 6 to 8), so I thought I'd add something there.

Reply:
I see. Thanks a lot!